From 2dda5cf427161ebb66a2de2429395d67fbba3955 Mon Sep 17 00:00:00 2001
From: Tomas Orsava <torsava@redhat.com>
Date: Tue, 26 Jan 2021 15:02:57 +0100
Subject: [PATCH] Skip flaky tests
---
 psutil/tests/test_system.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/psutil/tests/test_system.py b/psutil/tests/test_system.py
index a55164c..912a254 100755
--- a/psutil/tests/test_system.py
+++ b/psutil/tests/test_system.py
@@ -724,6 +724,7 @@ class TestNetAPIs(PsutilTestCase):
             self.assertEqual(psutil.net_io_counters(pernic=True), {})
             assert m.called
 
+    @unittest.skip("Skip flaky tests")
     def test_net_if_addrs(self):
         nics = psutil.net_if_addrs()
         assert nics, nics
-- 
2.29.2

